packages/custom-types/channel-data.d.ts

```diff
@@ -22,7 +22,7 @@ export default interface ChannelData {
   /** name of this channel */
   name: string,
   /** list of participants for this channel */
```

Comment:
I think the _correct_ route is for us to deprecate `ChannelData` - that will prevent us having to maintain it in this way.

Reply:
yep, I think so 👍